tox.ini

```diff
@@ -8,7 +8,7 @@ deps =
     -rrequirements-dev.txt
 commands =
     nosetests -v {posargs}
-    flake8 compose
+    flake8
```

Comment:
This can cause problems when you have extra files in the directory. If you were to create a another virtualenv it will try and check every file in that virtualenv.

I think this should be:

```
flake8 compose tests setup.py
```